roles/socle-config/files/releases.yaml

```
@@ -49,7 +49,7 @@ spec:
     # https://github.com/grafana/grafana/tags
     imageVersion: "10.4.3"
     # https://github.com/cloud-pi-native/helm-charts/tags
-    chartVersion: 1.4.0
+    chartVersion: 1.4.2
```

```suggestion
    chartVersion: ">=1.0.0 <2.0.0"
```
Et on fige si on le désire dans les dsc ?